elasticsearch/install.sh

```diff
@@ -15,7 +16,7 @@ export CLUSTER_NAME=placeholder
 mkdir -p ${HOME}
 ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
 /usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5
-/usr/share/elasticsearch/bin/plugin -i io.fabric8.elasticsearch/openshift-elasticsearch-plugin/0.7
+/usr/share/elasticsearch/bin/plugin -i io.fabric8.elasticsearch/openshift-elasticsearch-plugin/0.8
```

Comment:
since we just released version 0.9 of this plugin could you update to use that version instead?